onnxruntime/core/flatbuffers/ort.fbs

```diff
@@ -4,7 +4,7 @@
 namespace onnxruntime.experimental.fbs;
```

Comment:
Why experimental?

Reply:
This file format is experimental for now, there may be updates to the schema which may break backward compatibility, this will be removed if we eventually settle on an official schema.

Reply:
If a model with the fbs schema is shipped already and if the customers just want to update ORT for a bug fix, are they required to generate the model again if this schema changes? Might be ok for office who are in preview mode, but what about Edge? Probably can be handled by documentation and expectation management.

Reply:
They may have to. Not clear how it will need to change to support other EPs or things like NCHWc, so we've been very consistent in saying this is experimental and we're not promising backwards compatibility for the ORT format model at this point.

Reply:
For now (at least 1.5) we have a strict version requirement which the ORT version has to match the Version embedded in the ORT file